indico/web/assets/bundles.py

```diff
@@ -422,43 +422,10 @@ def rjs_bundle(name, *files):
 SASS_BASE_MODULES = ["sass/*.scss",
                      "sass/base/*.scss",
                      "sass/custom/*.scss",
-                     "sass/partials/*.scss"]
-
-
-def sass_module_bundle(module_name, depends=[]):
-    return Bundle('sass/modules/_{0}.scss'.format(module_name),
-                  filters=("pyscss", "cssrewrite", "cssmin"),
-                  output="sass/{0}_%(version)s.min.css".format(module_name),
-                  depends=SASS_BASE_MODULES + ['sass/modules/{0}/*.scss'.format(module_name)] + depends)
-
-agreements_sass = sass_module_bundle('agreements')
-attachments_sass = sass_module_bundle('attachments')
-contributions_sass = sass_module_bundle('contributions')
-abstracts_sass = sass_module_bundle('abstracts')
-papers_sass = sass_module_bundle('papers')
-reviewing_sass = sass_module_bundle('reviewing')
-tracks_sass = sass_module_bundle('tracks')
-registration_sass = sass_module_bundle('registrationform')
-payment_sass = sass_module_bundle('payment')
-roombooking_sass = sass_module_bundle('roombooking')
-dashboard_sass = sass_module_bundle('dashboard')
-category_sass = sass_module_bundle('category')
-category_management_sass = sass_module_bundle('category_management')
-admin_sass = sass_module_bundle('admin')
-bootstrap_sass = sass_module_bundle('bootstrap')
-
-eventservices_sass = sass_module_bundle('eventservices')
-event_display_sass = sass_module_bundle('event_display')
-event_management_sass = sass_module_bundle('event_management')
-overviews_sass = sass_module_bundle('overviews')
-vc_sass = sass_module_bundle('vc')
-news_sass = sass_module_bundle('news')
-users_sass = sass_module_bundle('users')
-auth_sass = sass_module_bundle('auth')
-surveys_sass = sass_module_bundle('surveys')
-timetable_sass = sass_module_bundle('timetable')
-sessions_sass = sass_module_bundle('sessions')
-markdown_sass = sass_module_bundle('markdown')
+                     "sass/partials/*.scss",
+                     "sass/modules/*.scss",
+                     "sass/modules/*/*.scss"]
```

Comment:
`sass/modules/**/*.scss` might cover both this and the previous one (`**` matches within the directory and any subdirectory). At least if webassets supports `**`